src/fairchem/core/trainers/base_trainer.py

```diff
@@ -273,7 +273,7 @@ def get_dataloader(self, dataset, sampler) -> DataLoader:
 
     def load_datasets(self) -> None:
         self.ocp_collater = OCPCollater(
-            self.config["model_attributes"].get("otf_graph", False)
+            self.config["model_attributes"].get("otf_graph", True)
```

Comment:
Is this line intentional? Does it change the default behavior for when otf_graph is not specified?